dune-project

```diff
@@ -19,7 +18,7 @@
   current_slack
   current_ssh
   ocluster-api
-  ; Opam dependencies
+  (current_ocluster (>= 0.3.0))
```

Comment:
Looks like there are some improvements that we could benefit from. OCluster could possibly use a release (not a dependency for merging this.)